lib/view_component/compiler.rb

```diff
@@ -125,9 +130,21 @@ def templates
 
         component_class._sidecar_files(extensions).each_with_object([]) do |path, memo|
           pieces = File.basename(path).split(".")
+
+          # if template is a localized file (ie file.country_code.html.erb)
+          # append into locale inside variant and fix regional dash (pt-BR, en-GB...)
+          if pieces.length >= 4
+            variant = [
+              pieces.third.split("+").second&.to_sym,
+              pieces.second.downcase.tr("-", "_")
+            ].compact.reduce { |s, x| "#{s}_#{x}" }
```

Comment:
Likely simpler as just
```suggestion
            variant = [
              pieces.third.split("+").second,
              pieces.second.underscore
            ].compact.join("_")
```

Does the result need to be a symbol?